utils/metrics/src/main/java/org/apache/edgent/metrics/MetricsSetup.java

```diff
@@ -111,6 +144,25 @@ private MBeanServer mbeanServer() {
         }
         return mBeanServer;
     }
+
+    /**
+     * Creates a default directory for metrics.
+     * 
+     * @return directory.getPath()
+     */
+    private String createDefaultDirectory() {
+        Path currentRelativePath = Paths.get("");
+        String pathMetrics = currentRelativePath.toAbsolutePath().toString() + FOLDER_METRICS;
+        File directory = new File(pathMetrics);
+        
+        if (!directory.mkdirs()) {
+            // Log: "Could not create the directory log"
```

Comment:
Can you fill in the logging?  Edgent uses slf4j for [logging](https://github.com/apache/incubator-edgent/blob/master/DEVELOPMENT.md#logging).  Sample use of it can be seen in [MqttConnector.java](https://github.com/apache/incubator-edgent/blob/master/connectors/mqtt/src/main/java/org/apache/edgent/connectors/mqtt/runtime/MqttConnector.java).  Thanks!